package.json

```diff
@@ -51,17 +52,16 @@
     "@khanacademy/eslint-plugin": "^2.0.0",
     "@khanacademy/wonder-stuff-testing": "^3.0.1",
     "@rollup/plugin-node-resolve": "^15.0.2",
-    "@storybook/addon-a11y": "^8.2.1",
-    "@storybook/addon-actions": "^8.2.1",
-    "@storybook/addon-designs": "^8.0.3",
-    "@storybook/addon-docs": "^8.2.1",
-    "@storybook/addon-essentials": "^8.2.1",
-    "@storybook/addon-interactions": "^8.2.1",
-    "@storybook/addon-mdx-gfm": "^8.2.1",
-    "@storybook/preview-api": "^8.2.1",
-    "@storybook/react": "^8.2.1",
-    "@storybook/react-vite": "^8.2.1",
-    "@storybook/test": "^8.2.1",
+    "@storybook/addon-a11y": "^8.5.0-alpha.10",
```

Comment:
`-alpha.10` 😱   

Are you planning to wait till this is released before landing? 😁

Reply:
hehe living on the edge! j/k

I can totally wait until it is officially released :). I mainly wanted to start this PR to be able to unlock their upcoming Accessibility addon (that uses this new testing tooling), so @marcysutton can try it out and give us some feedback, and also give feedback to the Storybook team as well, which she's planning to do :).